tests/pom.xml

```diff
@@ -981,5 +992,55 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>useRunningContainers</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>properties-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <phase>initialize</phase>
+                <goals>
+                  <goal>read-project-properties</goal>
+                </goals>
+                <configuration>
+                  <files>
+                    <file>${project.build.directory}/docker/auth.port.properties</file>
+                    <file>${project.build.directory}/docker/deviceregistry.port.properties</file>
+                    <file>${project.build.directory}/docker/broker.port.properties</file>
+                    <file>${project.build.directory}/docker/qpid.port.properties</file>
+                    <file>${project.build.directory}/docker/http.port.properties</file>
+                    <file>${project.build.directory}/docker/mqtt.port.properties</file>
+                    <file>${project.build.directory}/docker/adapter.amqp.port.properties</file>
```

Comment:
why is this file name prefixed with `adapter.` while the others aren't?

Reply:
That is in analogy to the naming of the maven properties (`mqtt.port`, `http.port, `adapter.amqp.port`).
I can have a look to make this more consistent overall.

Reply:
Does this mean that the file name represents the *base* name of the properties it contains? If so, then that is ok for me. I just wanted to understand ...

Reply:
> Does this mean that the file name represents the *base* name of the properties it contains? 

Yes, that was the idea.

Reply:
Taking a quick look at the maven-properties-plugin docs it doesn't look like this convention is required/enforced, is it? If not then I would propose that we use the same pattern for the endpoint property files across all adapters/components. WDYT?

Reply:
I've adapted the properties file names for the adapters now, having them all start with the "adapter." prefix. (The conformity to the maven property names is not that important here, I think.)